eth/handler_eth.go

```diff
@@ -204,3 +207,15 @@ func (h *ethHandler) handlePendingEtxs(pendingEtxs types.PendingEtxs) error {
 	}
 	return nil
 }
+
+func (h *ethHandler) handlePendingEtxsRollup(peer *eth.Peer, pEtxsRollup types.PendingEtxsRollup) error {
+	err := h.core.AddPendingEtxsRollup(pEtxsRollup)
+	if err != nil {
+		log.Error("Error in handling pendingEtxs rollup broadcast", "err", err)
```

Comment:
Should return err here.

Reply:
Added a IsValid check and returning error now